frontend-react/public/index.html

```diff
@@ -24,20 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+    <title>PAH FM</title>
   </head>
-  <body>
+  <body dir="rtl">
```

Comment:
Setting dir as an attribute of body is required in material-ui docs https://material-ui.com/guides/right-to-left/.

However, because it is set in `useLayoutEffect`, this hardcoded value can be removed.

Reply:
Thanks for the link :)